src/atomizer.js

```diff
@@ -235,7 +235,15 @@ module.exports = {
             minify: false,
             keepCamelCase: false,
             extCSS: '.css',
-            banner: ''
+            banner: '',
+            namespace: '#atomic',
+            rtl: false,
+            // TODO: Verify these are good defaults
```

Comment:
Hmmm, perhaps opening it up to allow arbitrary breakpoint naming?  Let's start tracking that in another issue.